contracts/liquidity_hub/pool-manager/src/tests/integration_tests.rs

```diff
@@ -2023,6 +2023,8 @@ mod swapping {
         );
     }
 
+    // TODO: make sure stableswap test works when pool has ONLY 2 assets
+
```

Comment:
Have tests for the stableswap with 2 assets being done? There are still some hardcoded 3s around, makes be feel it won't work with 2.

Reply:
it has been tested with `amp: 100`